doc/source/serve/deploy-many-models/multi-app.md

```diff
@@ -3,65 +3,66 @@
 
 In Ray 2.4+, deploying multiple independent Serve applications is supported. This user guide walks through how to generate a multi-application config file and deploy it using the Serve CLI, and monitor your applications using the CLI and the Ray Serve dashboard.
 
-## When to Use Multiple Applications
+## Context
+### Background 
 With the introduction of multi-application Serve, we walk you through the new concept of applications and when you should choose to deploy a single application versus multiple applications per cluster. 
 
 An application consists of one or more deployments. The deployments in an application are tied into a direct acyclic graph through [model composition](serve-model-composition). An application can be called via HTTP at the specified route prefix, and the ingress deployment handles all such inbound traffic. Due to the dependence between deployments in an application, one application is a unit of upgrade. 
 
-With that in mind, if you have many independent models each behind different endpoints, and you want to be able to easily add, delete, or update these models, then you should use multiple applications. Each model should then be deployed as a separate application. On the other hand, if you have ML logic and business logic distributed among separate deployments that all need to be executed for a single request, then you should use model composition to build a single application consisting of multiple deployments.
+### When to Use Multiple Applications
+Since one application is a unit of upgrade, having multiple applications allows you to deploy multiple models or groups of models that communicate with each other, while still being able to upgrade each application independently.
+
+If you have many independent models each behind different endpoints, and you want to be able to easily add, delete, or upgrade these models, then you should use multiple applications. Each model should then be deployed as a separate application. On the other hand, if you have ML logic and business logic distributed among separate deployments that all need to be executed for a single request, then you should use model composition to build a single application consisting of multiple deployments.
```

Comment:
The last line here seems a little contradictory to the paragraph above. I think calling one app from another should also be considered model composition, the real comparison is between having deployments in one app vs multiple. Maybe rephrase to say - 

Since one application is a unit of upgrade, having multiple applications allows you to deploy multiple models or groups of models. If you have many independent models each behind different endpoints, and you want to be able to easily add, delete, or upgrade these models, then you should use multiple applications. You can use in-place upgrades <link> to update applications independent of each other.

You can compose deployments across applications. However, if you have one repository with many models and/or business logic that need to be upgraded as a unit, we recommend having all those deployments in one application.

Reply:
I've reworded it a bit, lmk if it looks better!